src/core/ad-sizes.ts

```diff
@@ -388,7 +388,7 @@ const slotSizeMappings = {
 		phablet: [adSizes.outOfPage, adSizes.empty, adSizes.leaderboard],
 	},
 	'article-end': {
-		mobile: [adSizes.fluid],
+		mobile: [], // Mappings are dynamically added for this slot using additionalSizes
```

Comment:
very useful comment 👍

Reply:
Yes, it is otherwise it would be really confusing why we have an empty array!